test/unit/format.js

```diff
@@ -287,6 +287,14 @@ describe('format API', () => {
                     `[React Intl] No time format named: ${format}`
                 );
             });
+
+            it('should set default values', () => {
```

Comment:
I think it's worth a test to make sure if `hour`, `minute`, or `second` are specified, then the defaults are **not** applied.